api/apiroutes.go

```diff
@@ -21,11 +21,11 @@ import (
 	"time"
 
 	"github.com/decred/dcrd/chaincfg"
-	"github.com/decred/dcrd/dcrjson/v2"
-	"github.com/decred/dcrd/rpcclient/v2"
+	dcrjson "github.com/decred/dcrd/dcrjson/v2"
+	rpcclient "github.com/decred/dcrd/rpcclient/v2"
```

Comment:
Was a linter complaining?  The `/vN` suffix is an exception when modules are enabled.  No need explicitly name them.